src/lib/aws.ts

```diff
@@ -72,7 +79,9 @@ export async function registerECSTaskDefinition(
         requiresCompatibilities: baseTaskDefinition.requiresCompatibilities,
         tags: tags,
     })
-    return await client.send(command)
+    const result = await client.send(command)
+    console.log('AWS: RegisterTaskDefinitionCommand finished')
```

Comment:
Are you able to get the taskDefinition string at this point? It might help matching subsequent log messages when `getECSTaskDefinition` is called